Mergin/plugin.py

```diff
@@ -295,8 +338,22 @@ def create_new_project(self):
             return
 
         user_info = self.mc.user_info()
+        try:
+            workspaces = self.mc.workspaces_list()
+            if not workspaces:
+                self.show_no_workspaces_dialog()
+                self.current_workspace = None
+                return
+        except (URLError, ClientError) as e:
+            ns = self.mc.global_namespace()
```

Comment:
this only exists for CE server, no?

Reply:
Yes, and should be removed with the upcoming API, since the global namespace will be returned by user info and workspaces endpoint. Then we will not distinguish between ce and ee